build.gradle

```diff
@@ -153,7 +158,22 @@ task renameShadowJar(type: Copy, group: 'release', dependsOn: [shadowJar]) {
 task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
+task prepareGameEngineProperties(group: 'release') {
```

Comment:
A comment here with some context on why we are doing this operation would perhaps be very useful for current/future reference. I don't quite rightly know offhand why this copy would happen.